bang/magicollections.py

```diff
@@ -1947,22 +1940,18 @@ def get_queryset(self, queryset, parameters, request):
 
         def to_fields(self, item, *args, **kwargs):
             fields = super(SongCollection.ItemView, self).to_fields(item, *args, **kwargs)
-            note_image = u'{}img/note.png'.format(RAW_CONTEXT['static_url'])
             for difficulty, verbose_name in models.Song.DIFFICULTIES:
-                diff_value=''
+                diff_value = ''
                 diff = getattr(item, u'{}_difficulty'.format(difficulty), None)
-                if diff != None: diff_value=mark_safe(u'{big_images}{small_images}'.format(diff=diff,
-                    big_images=(u'<img src="{}" class="song-big-note">'.format(note_image) * (diff // 5)),
-                    small_images=(u'<img src="{}" class="song-small-note">'.format(note_image) * (diff % 5)),
-                    ))
-                if diff_value != '': diff_value+='<br />'
+                if diff is not None:
+                    diff_value = generateDifficulty(diff, True)
                 if getattr(item, u'{}_notes'.format(difficulty), None) != None: diff_value+=_(u'{} notes').format(getattr(item, u'{}_notes'.format(difficulty), None))
                 if diff_value != '':
                     fields[difficulty] = {
                         'verbose_name': verbose_name,
                         'type': 'html',
                         'value': diff_value,
-                        'image': u'{static_url}img/songs/{difficulty}.png'.format(static_url=RAW_CONTEXT['static_url'], difficulty=difficulty,),
+                        'image': staticImageURL('songs/{difficulty}.png'.format(difficulty=difficulty)),
```

Comment:
`staticImageURL` is also here to make it easier and allow you to avoid formatting yourself, so you can do:

```python
staticImageURL(difficulty, folder='songs', extension='png')
```